src/shogun/kernel/ShiftInvariantKernel.h

```diff
@@ -135,10 +115,8 @@ class ShiftInvariantKernel: public Kernel
 	std::shared_ptr<Distance> m_distance;
 
 private:
-	/** Registers the parameters (serialization support). */
 	virtual void register_params();
 
-	/** Precomputed distance instance */
```

Comment:
Could you in the future not do all these changes.... this is supposed to be a simple addition with a few lines diff. Removing all the docstrings just causes noise. If you want to do it, simply make another PR in the future